awscli/customizations/configure.py

```diff
@@ -302,6 +303,9 @@ def _run_main(self, args, parsed_globals):
         region = self._lookup_config('region')
         self._display_config_value(region, 'region')
 
+        ca_bundle = self._lookup_config('ca_bundle')
```

Comment:
Fyi, this comment didn't get outdated because this seemed like the wrong place to achieve the desired effect. Removing the lines here means that `aws configure list` won't print the line with the `ca_bundle` settings. I didn't think that was what you meant. Instead, I removed the lines [here](https://github.com/lorengordon/aws-cli/commit/b6d1dc2d06a3d1e78dfddd6ea64a8a8620fe1b11#diff-67fa1a65752518af4d764aa3e6a9cf31L551) so `aws configure` wouldn't prompt for `ca_bundle`.